flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/test/java/org/apache/flink/cdc/connectors/base/experimental/config/MySqlSourceConfigFactory.java

```diff
@@ -63,6 +63,10 @@ public MySqlSourceConfig create(int subtaskId) {
         props.setProperty("database.user", checkNotNull(username));
         props.setProperty("database.password", checkNotNull(password));
         props.setProperty("database.port", String.valueOf(port));
+        // props.setProperty("hostname", checkNotNull(hostname));
+        // props.setProperty("user", checkNotNull(username));
+        // props.setProperty("password", checkNotNull(password));
+        // props.setProperty("port", String.valueOf(port));
```

Comment:
Why do we need to change this part?

Reply:
These comments are useless, I will remove them